examples/howto/sbi_hnncore_tutorial.py

```diff
@@ -0,0 +1,198 @@
+"""
+==========================================
+09. SBI with HNN-core: Parameter Inference
+==========================================
+
+This tutorial demonstrates how to use Simulation-Based Inference (SBI)
+with HNN-core to infer network parameters from simulated data. We'll
+simulate neural network activity, then use SBI to infer the parameters
+that generated this activity.
+"""
+
+# Authors: Abdul Samad Siddiqui <abdulsamadsid1@gmail.com>
+#          Nick Tolley <nicholas_tolley@brown.edu>
+#          Ryan Thorpe <ryan_thorpe@brown.edu>
+#          Mainak Jas <mjas@mgh.harvard.edu>
+
+###############################################################################
+# First, we need to import the necessary packages. The Simulation-Based
+# Inference (SBI) package can be installed with ``pip install sbi``.
+# Let us import ``hnn_core`` and all the necessary libraries.
+
+import numpy as np
+import torch
+from hnn_core.batch_simulate import BatchSimulate
+from sbi import utils as sbi_utils
+from sbi import inference as sbi_inference
+import matplotlib.pyplot as plt
+from hnn_core.network_models import jones_2009_model
+
+###############################################################################
+# Now, we'll set up our simulation parameters. We're using a small number of
+# simulations for this example, but in practice, you might want to use more.
+
+n_simulations = 100
+n_jobs = 20
+
+###############################################################################
+# This function sets the parameters for our neural network model. It adds an
+# evoked drive to the network with specific weights and delays.
+#
+# The `add_evoked_drive` function simulates external input to the network,
+# mimicking sensory stimulation or other external events.
+#
+# - `evprox` indicates a proximal drive, targeting dendrites near the cell
+#   bodies.
+# - `mu=40` and `sigma=5` define the timing (mean and spread) of the input.
+# - `numspikes=1` means it's a single, brief stimulation.
```

Comment:
```suggestion
```

I think we can omit this description. I also wouldn't call this a "single brief stimulation". The number of spikes is more referring to how many spikes the presynaptic neurons are firing, the "briefness" of it would depend on if they are all firing synchronously, or if there is only one neuron connected to all targets (happens when `cell_specific=True`)